pkg/goformat/runner.go

```diff
@@ -0,0 +1,166 @@
+package goformat
```

Comment:
I propose renaming the package to `fmt`. Functions from this package are used only in `pkg/commands/fmt.go`, so it will be okay. Additionally, this avoids a possible collision with the `goformatters` package.